README.md

```diff
@@ -200,7 +200,7 @@ Shoes 4 is developed in a TDD style using RSpec. You should be writing and runni
 
 The simplest way to do this is with rake tasks.
 
-    $ rake spec
+    $ rake
```

Comment:
I'm not sure, maybe you fixed it and I just haven't knowingly tried afterwards ever again :)